File: InteracoesMedicamentosas/Models/Interacao.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace InteracoesMedicamentosas.Models
{
    public class Interacao
    {
        public int InteracaoId { get; set; }

        public int ReacaoId { get; set; }
        public int ProdutoId { get; set; }

        public virtual Reacao Reacao { get; set; }
        public virtual Produto Produto { get; set; }
    }
}

File: InteracoesMedicamentosas/scripts/JS/Interacao.js
function SalvarInteracao() {

    //debugger;

    //Produto
    var produto = $("#Produto").val();

    //Cliente
    var reacao = $("#Reacao").val();


    //Gravar
    var url = "/Interacao/Create";

    $.ajax({
        url: url
        , type: "POST"
        , datatype: "json"
        , data: { InteracaoId: 0, ReacaoId: reacao, ProdutoId: produto }
        , success: function (data) {
            debugger;
            if (data.Resultado > 0) {
                ListarItens(data.Resultado);
            }
        }
    });
}

function ListarItens(idPedido) {

    var url = "/Itens/ListarItens";

    $.ajax({
        url: url
        , type: "GET"
        , data: { id: idPedido }
        , datatype: "html"
        , success: function (data) {
            var divItens = $("#divItens");
            divItens.empty();
            divItens.show();
            divItens.html(data);
        }
    });

}